.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,59 @@
+name: Build artifact
+
+on:
+  push:
+    paths:
+      - 'plugin/**'
```

Comment:
Can we add
```'.github/workflows/build-artifact.yml'```
to this list and the pull_request paths list as well?

Reply:
Okay!